virt/vm_networking/virt-accessing-vm-internal-fqdn.adoc

```diff
@@ -0,0 +1,29 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-accessing-vm-internal-fqdn"]                            
+= Accessing a virtual machine  by using its internal FQDN                                             
+include::_attributes/common-attributes.adoc[]                   
+:context: virt-accessing-vm-internal-fqdn                        
+                                                                
+toc::[]         
+
+You can access a virtual machine (VM) that is connected to the default internal pod network, on a stable fully qualified domain name (FQDN), by using headless services.
+
+A Kubernetes _headless service_ is a form of service that does not allocate a cluster IP address to represent a set of pods. Instead of providing a single virtual IP address for the service, a headless service creates a DNS record for each pod associated with the service. You can expose a VM through its FQDN without the need to expose a specific TCP or UDP port. 
+ 
+[NOTE]
+====
+If you created a VM by using the {product-title} web console, you can find its internal FQDN listed in the *Network* tile on the *Overview* tab of the *VirtualMachine details* page. See xref:../../virt/vm_networking/virt-accessing-vm-internal-fqdn.adoc#virt-connecting-vm-internal-fqdn_virt-accessing-vm-internal-fqdn[Connecting to a virtual machine by using its internal FQDN] for next steps to access the VM.
```

Comment:
```suggestion
If you created a VM by using the {product-title} web console, you can find its internal FQDN listed in the *Network* tile on the *Overview* tab of the *VirtualMachine details* page. For more information, see xref:../../virt/vm_networking/virt-accessing-vm-internal-fqdn.adoc#virt-connecting-vm-internal-fqdn_virt-accessing-vm-internal-fqdn[Connecting to a virtual machine by using its internal FQDN].
```

Reply:
Accessing the VM is the next procedure that the user has to jump to if they already have the FQDN from the web console. I think the wording makes more sense as is.